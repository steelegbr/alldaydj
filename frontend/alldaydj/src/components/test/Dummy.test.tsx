import React from 'react'
import { shallow } from 'enzyme'
import Dummy from './Dummy'

describe('dummy component', () => {
  function mountComponent () {
    return shallow(<Dummy />)
  }

  it('snapshot', () => {
    const dummy = mountComponent()
    expect(dummy).toMatchSnapshot()
  })
})
