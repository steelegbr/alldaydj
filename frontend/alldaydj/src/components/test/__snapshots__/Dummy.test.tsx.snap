// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dummy component snapshot 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Typography))
    variant="h1"
  >
    Hello from AllDay DJ
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    paragraph={true}
  >
    This is a stand-in component to test routing, etc.
  </WithStyles(ForwardRef(Typography))>
</Fragment>
`;
